static/js/common.js

```diff
@@ -37,13 +37,14 @@ exports.password_quality = function (password, bar, password_field) {
 
     // Compute a quality score in [0,1].
     var result  = zxcvbn(password);
-    var quality = Math.min(1,Math.log(1 + result.crack_times_seconds.
+    var bar_progress = Math.min(1,Math.log(1 + result.crack_times_seconds.
                                           offline_slow_hashing_1e4_per_second) / 22);
+    var quality = result.guesses_log10 * Math.log(10, 2);
```

Comment:
Let's call this `entropy_bits` -- that makes it very clear what it means, and what the units are.

Reply:
Similarly `min_quality` should be `min_entropy_bits`, and the field data it comes from should be `minEntropyBits`.